### DISCUSSION TITLE
TheThingsNetwork (TTN) - Integration

### DISCUSSION THREAD

**Discussion by CK1-CK** [sandeepmistry/arduino-LoRa#567] [Q&A]
Hello,

I would like to use your Lib to transfer data from a door switch to TheThingsNetwork (TNN).

At first glance, it seems quite simple. Unfortunately, I don't know how to integrate DevEUI, NwkSKey, SNwkSIntKey, NwkSEncKey and AppSKey. 
Apparently something has also changed with TTN (version jump from 2.0 to 3.0) and most tutorials only show the old variant.

**My hardware**:
-Arduino Mini Pro 3.3V 8Mhz
-Lora Board with SX1276

Is there an easy way to connect to TTN with the ardunio-Lora Lib?

best regards
CK1

**Comment by Kongduino** [comment]
No. This is a LoRa P2P-only library. You could use something like the Beelan library for LoRaWAN.
